pages/support.html

```diff
@@ -0,0 +1,34 @@
+---
+permalink: /support
+layout: master
+---
+
+<div class="flex flex-col justify-center w-full text-white text-2xl space-y-8">
+    <div class="flex justify-center">
+        <p class="text-8xl font-extrabold">
+            Support Blue Fire 💙
+        </p>
+    </div>
+    <p>
+        All our projects are community driven and open source.
+    </p>
+    <p>
+        We love contributions in the form of meaningful PRs, assistance with projects, participation on <a
+            href="https://discord.gg/pxrBmy4">our Discord server</a>, and just spreading the word and the love within
```

Comment:
```suggestion
            href="https://discord.gg/pxrBmy4">our Discord server</a>, and just spreading the word within
```
Somehow sounds more professional? And also avoids repeating love two times in the sentence